CHANGELOG.md

```diff
@@ -2,11 +2,11 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
-## 1.0.2 - TBD
+## 1.1.0 - TBD
 
 ### Added
 
-- Nothing.
+- [#3](https://github.com/zendframework/zend-config-aggregator-modulemanager/pull/3) adds support for `ViewHelperProviderInterface`
```

Comment:
I am not sure if it should be considered as "new feature" or a bugfix.
Also description should be a bit extended, to describe what is the fix/feature.

Reply:
Actually it is a "new feature" as the `view_helpers` array key didnt exist in the first place.

It would be a bug if we say "we are handling *every* method in your module" but actually we just had the features provided by the current `ConfigProvider`. 
I am not sure if its a bugfix as the package never said that it can parse the `view_helpers` config at all.